### PR TITLE
[tw] Localize 'Datacenter' for Traditional Chinese

### LINKED ISSUE #2567: [tw] Localize Datacenter for Traditional Chinese
- Localize `Datacenter` for Traditional Chinese
- Source: https://glossary.cncf.io/data-center/

### PR DESCRIPTION
**Pull request by KevinSpringfield**
### Describe your changes

1. Localize Datacenter for Traditional Chinese
2. Source: https://glossary.cncf.io/data-center/

### Related issue number or link (ex: `resolves #issue-number`)
Resolves #2567 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-tw/data-center.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+tags: ["基礎設施", "基本原理", ""]
+---
+
+資料中心是一種被設計來存放電腦 （通常是伺服器） 的特殊建築或設施。
+這些資料中心往往連接到高速網路的線路上，特別是專門用於[雲端運算](/zh-tw/cloud-computing/)的資料中心。
+資料中心所在的建築具備了即使在不理想的情況下也能維持服務的設備，包含在停電期間提供電力的發電機以及冷卻發熱中計算機的強大空調設備。
+
+## 解決的問題 {#problem-it-addresses}
+
+在 1990 年代後期資料中心變得普及之前，市面上主要是用於處理特定任務或個人用來完成工作的電腦。
```
Comment (johnlinp) [inline content/zh-tw/data-center.md:14]:
```suggestion
在 1990 年代後期，資料中心變得普及之前，市面上主要是用於處理特定任務或個人用來完成工作的電腦。
```

File: content/zh-tw/data-center.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+但是電腦具有有限的資源 （硬碟、RAM 以及 CPU）。
```
Comment (johnlinp) [inline content/zh-tw/data-center.md:16]:
```suggestion
但是電腦只有有限的資源 （硬碟、RAM 以及 CPU）。
```

File: content/zh-tw/data-center.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+這意味著在電腦上執行應用程式有著和電腦相同的硬性限制，侷限了可執行的應用程式種類。
+在資料中心問世前，應用程式的規模受限於其執行電腦的容量。
+但如果考慮到像是 Gmail 或 Netflix 等規模的應用程式 （應用程式，並非在手機或電腦上的使用者介面），它們需要超過一台電腦能提供的運算容量，而這正是資料中心發揮作用的地方。
```
Comment (johnlinp) [inline content/zh-tw/data-center.md:19]:
```suggestion
但如果考慮到像是 Gmail 或 Netflix 等規模的應用程式 （這邊指的是整個應用程式，並非只是在手機或電腦上的使用者介面），它們需要超過一台電腦能提供的運算容量，而這正是資料中心發揮作用的地方。
```

File: content/zh-tw/data-center.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+## 如何幫助我們 {#how-it-helps}
+
+藉由連接各種伺服器，用戶可以建立一套像是「超級電腦」的[分散式系統](/zh-tw/distributed-systems/)。
```
Comment (johnlinp) [inline content/zh-tw/data-center.md:23]:
```suggestion
藉由連接各種伺服器，使用者可以建立一套像是「超級電腦」的[分散式系統](/zh-tw/distributed-systems/)。
```

File: content/zh-tw/data-center.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+因為結合了多台機器的運算能力，我們現在能夠執行更龐大的應用程式或是處理更需要強大運算能力的任務。
+資料中心強化了大多數我們日常中使用的應用程式。
+
+[公有雲](/zh-tw/cloud-computing)就是出借其運算容量給用戶的資料中心。
```
Comment (johnlinp) [inline content/zh-tw/data-center.md:27]:
```suggestion
[公有雲](/zh-tw/cloud-computing)就是出借其運算容量給使用者的資料中心。
```

**Review by johnlinp — COMMENTED** [review]
感謝翻譯，我這邊只有幾個語句上通順度的建議。

File: content/zh-tw/data-center.md (hunk 5 above)
Comment (KevinSpringfield) [reply content/zh-tw/data-center.md]: 已修改

File: content/zh-tw/data-center.md (hunk 4 above)
Comment (KevinSpringfield) [reply content/zh-tw/data-center.md]: 已修改

File: content/zh-tw/data-center.md (hunk 3 above)
Comment (KevinSpringfield) [reply content/zh-tw/data-center.md]: 已修改

File: content/zh-tw/data-center.md (hunk 2 above)
Comment (KevinSpringfield) [reply content/zh-tw/data-center.md]: 已修改

File: content/zh-tw/data-center.md (hunk 1 above)
Comment (KevinSpringfield) [reply content/zh-tw/data-center.md]: 已修改

**Review by johnlinp — APPROVED** [review]
感謝!

File: content/zh-tw/data-center.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+資料中心所在的建築具備了即使在不理想的情況下也能維持服務的設備，包含在停電期間提供電力的發電機以及冷卻發熱中計算機的強大空調設備。
+
+## 解決的問題 {#problem-it-addresses}
+
+在 1990 年代後期，資料中心變得普及之前，市面上主要是用於處理特定任務或個人用來完成工作的電腦。
+
+但是電腦只有有限的資源 （硬碟、RAM 以及 CPU）。
+這意味著在電腦上執行應用程式有著和電腦相同的硬性限制，侷限了可執行的應用程式種類。
+在資料中心問世前，應用程式的規模受限於其執行電腦的容量。
+但如果考慮到像是 Gmail 或 Netflix 等規模的應用程式 （這邊指的是整個應用程式，並非只是在手機或電腦上的使用者介面），它們需要超過一台電腦能提供的運算容量，而這正是資料中心發揮作用的地方。
```
Comment (ydFu) [inline content/zh-tw/data-center.md:19]:
```suggestion
但如果考慮到像是 Gmail 或 Netflix 等規模的應用程式 （這邊指的是整個應用程式，並非只是在手機或電腦上的使用者介面），任何單一電腦的計算能力皆無法提供上述應用程式所需的計算能力，而這正是資料中心發揮作用的地方。
```

File: content/zh-tw/data-center.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+## 如何幫助我們 {#how-it-helps}
+
+藉由連接各種伺服器，使用者可以建立一套像是「超級電腦」的[分散式系統](/zh-tw/distributed-systems/)。
+因為結合了多台機器的運算能力，我們現在能夠執行更龐大的應用程式或是處理更需要強大運算能力的任務。
+資料中心強化了大多數我們日常中使用的應用程式。
+
+[公有雲](/zh-tw/cloud-computing)就是出借其運算容量給使用者的資料中心。
+在過去幾年間，我們已經看到企業逐漸將原有的資料中心遷移至雲端。
```
Comment (ydFu) [inline content/zh-tw/data-center.md:28]:
```suggestion
在過去幾年間，我們已經看到企業逐漸將自有的資料中心遷移至雲端。
```

File: content/zh-tw/data-center.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+---
+title: 資料中心
+status: Completed
+category: 技術
+tags: ["基礎設施", "基本原理", ""]
+---
+
+資料中心是一種被設計來存放電腦 （通常是伺服器） 的特殊建築或設施。
```
Comment (ydFu) [inline content/zh-tw/data-center.md:8]:
```suggestion
資料中心是一種被設計來存放電腦（通常是伺服器）的特殊建築或設施。
```

File: content/zh-tw/data-center.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+這些資料中心往往連接到高速網路的線路上，特別是專門用於[雲端運算](/zh-tw/cloud-computing/)的資料中心。
+資料中心所在的建築具備了即使在不理想的情況下也能維持服務的設備，包含在停電期間提供電力的發電機以及冷卻發熱中計算機的強大空調設備。
+
+## 解決的問題 {#problem-it-addresses}
+
+在 1990 年代後期，資料中心變得普及之前，市面上主要是用於處理特定任務或個人用來完成工作的電腦。
+
+但是電腦只有有限的資源 （硬碟、RAM 以及 CPU）。
```
Comment (ydFu) [inline content/zh-tw/data-center.md:16]:
```suggestion
但是電腦只有有限的資源（硬碟、RAM 以及 CPU）。
```

Comment (KevinSpringfield) [reply content/zh-tw/data-center.md]: 已修正

File: content/zh-tw/data-center.md (hunk 8 above)
Comment (KevinSpringfield) [reply content/zh-tw/data-center.md]: 已修正

File: content/zh-tw/data-center.md (hunk 7 above)
Comment (KevinSpringfield) [reply content/zh-tw/data-center.md]: 已修正

File: content/zh-tw/data-center.md (hunk 6 above)
Comment (KevinSpringfield) [reply content/zh-tw/data-center.md]: 已修正

**Review by pichuang — APPROVED** [review]
LGTM

**Pull request merged**